.github/workflows/gemini-issue-automated-triage.yml

```diff
@@ -55,9 +55,9 @@ jobs:
           app-id: '${{ vars.APP_ID }}'
           private-key: '${{ secrets.APP_PRIVATE_KEY }}'
 
-      - name: 'Run Gemini Issue Triage'
+      - name: 'Suggest Labels'
         uses: './'
-        id: 'gemini_issue_triage'
+        id: 'suggest_labels'
         env:
           GITHUB_TOKEN: '${{ steps.generate_token.outputs.token || secrets.GITHUB_TOKEN }}'
```

Comment:
can we remove this too by fetching labels upfront? then we can also remove `gh label list`